adhocteam/Head-Start-TTADP
# TTAHUB-86 Hide the Report Approved Section when Printing

## Description of change

The purpose of this JIRA is to hide the Reports Approved section along with both Creator / Manager notes.

Per JIRA comments these sections should NEVER be contained in the print out.

## How to test

Login to the application and view a Approved report as BOTH the Submitter and Approver. When you go to print the AR via the browser print both notes sections should be missing in the print out.


## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-86


## Checklists

### Every PR


~- [ ] API Documentation updated~
~- [ ] Boundary diagram updated~
~- [ ] Logical Data Model updated~
~- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions~

### Production Deploy

~- [ ] Staging smoke test completed~

### After merge/deploy
